app/views/users/show.html.erb

```diff
@@ -1 +1,22 @@
-<h1>This is the user#show view</h1>
\ No newline at end of file
+<div>
+  <img src='https://unsplash.com/photos/F_-0BxGuVvo' alt="user_image">
+</div>
+
+<div>
+  <p><%= @user.name %></p>
```

Comment:
Kindly follow [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_page.png) wireframe for the user show page. There should be a button name `See all posts at the end of the page.